R/dt_summary.R

```diff
@@ -251,7 +251,13 @@ dt_summary_build <- function(data,
               context = context
             )
 
-          formatter(x)
+          formatted <- formatter(x)
```

Comment:
As currently written, the options from the main gt table (locale, etc.) won't be respected by this formatter call, because we're using a temporary gt table.

Reply:
Now changed.